swagger/v1/person.json

```diff
@@ -42,6 +42,13 @@
             "example": "1965-10-24",
             "description": "Person's date of birth in ISO 8601 format"
           },
+          "image_url": {
+            "type": "string",
+            "format": "uri",
+            "example": "http://localhost:3000/api/v1/people/88089bbd-8719-4192-b309-f9db9105d3e1/image.jpg",
+            "description": "URL where image of person can be retrieved",
```

Comment:
Is the description right?
Does it need the article:
`URL where *the* image of person can be retrieved`
or maybe we can be more descriptive:
`URL pointing to the image of the person`

Up to you.